editor/components/inserter/index.js

```diff
@@ -51,8 +56,9 @@ class Inserter extends wp.element.Component {
 					onClick={ this.toggle }
 					className="editor-inserter__toggle"
 					aria-haspopup="true"
+					buttonRef={ ( node ) => this.nodes.toggle = node }
```

Comment:
small preference for `this.toggleNode` but this is minor. Feel free to keep as is.